doc/reference/autoinstall-reference.rst

```diff
@@ -144,8 +144,8 @@ installed will not have a keyboard at all in which case the value used here
 does not matter.
 
 The mapping's keys correspond to settings in the :file:`/etc/default/keyboard`
-configuration file. See
-`its manual page <http://manpages.ubuntu.com/manpages/keyboard.5.html>`_
+configuration file. See the
+`keyboard(5) manual page <https://manpages.ubuntu.com/manpages/lunar/en/man5/keyboard.5.html>`_
```

Comment:
Changing http for https is a great change. However, I feel a bit uncomfortable about hardcoding lunar here. The redirects are expected to change over time to point to newer Ubuntu releases ; which means we would need to manually update the links.

Reply:
> I feel a bit uncomfortable about hardcoding lunar here

Good point. Unfortunately, I don't know a good workaround (yet). I'll be looking into a Sphinx extension that allows the use of variables (rST substitutions) within URLs, which would allow us to have something like `https://manpages.ubuntu.com/manpages/|distro_version|/en/man5/keyboard.5.html`. Until then, I'm afraid it needs to be manual.

Reply:
Okay, I guess that's acceptable.